test/integration/testdata/mysql.yaml

```diff
@@ -26,6 +26,10 @@ spec:
       containers:
       - image: mysql:5.6
```

Comment:
consider updating the mysql to a newer version maybe better performance ?

Reply:
yuh, I was thinking about that. We could use mysql:5.7 instead of mysql:5.6